scala/build/maintenance.Dockerfile

```
@@ -256,6 +256,10 @@ ENV PATH /home/qa/.local/bin:/home/qa/sbt/bin:/home/qa/bin:${PATH}
 RUN pipx install awscli
 # RUN ln -s /home/qa/.local/bin/aws /usr/local/bin/aws
 
+# trcli
+RUN pip install --upgrade pip pipx
```

Hadolint

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)